src/v/kafka/client/client.cc

```diff
@@ -263,4 +264,15 @@ ss::future<offset_commit_response> client::consumer_offset_commit(
       });
 }
 
+ss::future<kafka::fetch_response> client::consumer_fetch(
+  const group_id& g_id,
```

Comment:
take these primitives by value?

Reply:
ah they are strings, not integers.

Reply:
Yes, typically stringified UUID. I'm happy to take a copy and move, but despite being futurized, it's currently a synchronous operation. It follows the pattern of all the other consumer group endpoints.